jipdate.py

```diff
@@ -277,11 +281,20 @@ def parse_status_file(jira, filename):
     Jira call. This for example removes the beginning until it finds a
     standalone [ISSUE] tag. It will also remove all comments prefixed with '#'.
     """
+    global g_args
+
     # Regexp to match Jira issue on a single line, i.e:
     # [SWG-28]
     # [LITE-32]
-    # etc ...
-    regex = r"^\[([A-Z]*-[0-9]*).*\]\n$"
+    # ...
+    regex = r"^\[([A-Z]+-[0-9]+).*\]\n$"
+
+    # Regexp to match a tag that indicates we should stop processing, ex:
+    # [STOP]
+    # [JIPDATE-STOP]
+    # [OTHER]
```

Comment:
I've force-pushed and updated this pull request so that now it should support it, yes.